localstack/services/awslambda/lambda_executors.py

```diff
@@ -1299,12 +1309,18 @@ def _execute_in_custom_runtime(
                 log_output += "\n%s" % additional_logs
 
         func_arn = lambda_function and lambda_function.arn()
-        log_lambda_result(func_arn, result, log_output)
+        output = OutputLog(result, log_output)
+        LOG.debug(
+            f"Lambda {func_arn} result / log output:"
+            f"\n{output.stdout_formatted()}"
+            f"\n>{output.stderr_formatted()}"
+        )
 
         # store log output - TODO get live logs from `process` above?
         # store_lambda_logs(lambda_function, log_output)
 
         if return_code != 0:
```

Comment:
even if the truncation happens (stdout), we can write the full log into a temporary file

Reply:
But we only do so if there was an error, right? I guess we could make this configurable? Not important for now though.

Reply:
Only on errors yes. Perhaps don't see that as a requirement tho.